.pipeline/config.yml

```diff
@@ -2,14 +2,6 @@
 # This file configures the SAP Cloud SDK Continuous Delivery pipeline of your project.
 # For a reference of the configuration concept and available options, please have a look into its documentation.
 #
-# The documentation for the most recent pipeline version can always be found at:
```

Comment:
[q] @marikaner do we want to keep the pipeline or is this something we do not want to promote anymore?

Reply:
@FrankEssenberger As I discussed with Marika, we just remove all references to the pipeline and AP Cloud SDK Continuous Delivery Toolkit. I just missed this part. I remove this.
@marikaner please double-check for these updates.